Add communications and outreach information

choldgraf opened the pull request:
This adds documentation about our communications and outreach channels, with the goal of making out strategy and channels of communication easier for others to understand and use.

- Defines the various accounts that we use and how to access them / request for access
- Defines some guidelines for what kinds of things to post
- Defines some checklists and steps to take to lower the decision-cost of posting
- Moves this out of the "Community Guidance" section into its own dedicated section.

closes https://github.com/2i2c-org/meta/issues/193

damianavila commented on communication/blog.md:
```diff
@@ -0,0 +1,43 @@
+    * Major deliverables that we’ve met on a project
+    * New collaborations
+* **Open source updates**. If we do particular service work, or make a strong technical improvement in an open source project, we should tell the world about it. 
+* **Updates from the hubs pilot**. As our pilot hub infrastructure grows and evolves, and as we serve more communities with it, we should tell others about the impact we are having.
```
Maybe we should not keep using the pilot hub concept? I think we have removed it from other places already...

choldgraf commented:
I believe that I've addressed all of your questions @damianavila ! I've also added "how to access" information for each of the accounts there (and if you / others would like access please just let me know!)

damianavila commented:
> I've also added "how to access" information for each of the accounts there 

Super, thanks!

> (and if you / others would like access please just let me know!)

Yes, I would like to get access to them (to further ANN the job posting).

choldgraf commented:
FYI - I've:

- Added @damianavila as a "Super Admin" on our LinkedIn.
- Given him the password to our Twitter account

choldgraf commented:
ok will merge this in as well, since I think it mostly just makes a bunch of information available that was not available before, rather than defining new policy!

choldgraf commented:
@damianavila yep, see Slack
